Description: Make sure file systems can be fixed on machines with locked root accounts (as Ubuntu does by default). Don't require --force for sulogin.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug-Debian: https://bugs.debian.org/326678

Index: util-linux-2.37.2-4ubuntu1/login-utils/sulogin.c
===================================================================
--- util-linux-2.37.2-4ubuntu1.orig/login-utils/sulogin.c
+++ util-linux-2.37.2-4ubuntu1/login-utils/sulogin.c
@@ -991,7 +991,8 @@ int main(int argc, char **argv)
 				const char *passwd = pwd->pw_passwd;
 				char *answer;
 				int doshell = 0;
-				int deny = !opt_e && locked_account_password(pwd->pw_passwd);
+				/* Ubuntu's root account is locked by default allow access without --force */
+				int deny = 0;
 
 				doprompt(passwd, con, deny);
 
